### PR TITLE
Adding Target identities upon MobileCore.getSDKIdentities API call

### PR DESCRIPTION
**Pull request by swarna04**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by praveek — APPROVED** [review]

File: AEPIdentity/Sources/MobileIdentities.swift [hunk 1]
```diff
@@ -82,12 +85,13 @@ struct MobileIdentities: Codable {
         let configurationStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.CONFIGURATION, event)?.status ?? .none
         let analyticsStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.ANALYTICS, event)?.status ?? .none
         let audienceStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.AUDIENCE, event)?.status ?? .none
-        // TODO: Target
+        let targetStatus = sharedStateProvider(IdentityConstants.SharedStateKeys.AUDIENCE, event)?.status ?? .none
```
Comment (sbenedicadb) [inline AEPIdentity/Sources/MobileIdentities.swift:88]: this is meant to be `IdentityConstants.SharedStateKeys.TARGET`, yes?

File: AEPIdentity/Sources/MobileIdentities.swift [hunk 2]
```diff
@@ -49,7 +49,10 @@ struct MobileIdentities: Codable {
     static let NAMESPACE_ANALYTICS_AID = "AVID"
     static let NAMESPACE_USERIDENTIFIER = "vid"
     static let NAMESPACE_AUDIENCE_UUID = "0"
+    static let NAMESPACE_TARGET_TNTID = "tntid"
```
Comment (sbenedicadb) [inline AEPIdentity/Sources/MobileIdentities.swift:52]: nit: we have `TNT_ID` and `THIRD_PARTY_ID` constants in `IdentityConstants` but `...TNTID` and `...THIRDPARTYID` in this file.

let's get consistent about whether these should have underscores or not :) 

(my vote is yes)

**Review by sbenedicadb — CHANGES_REQUESTED** [review]

File: AEPIdentity/Sources/MobileIdentities.swift (hunk 1 above)
Comment (swarna04) [reply AEPIdentity/Sources/MobileIdentities.swift:88]: Good catch!

File: AEPIdentity/Sources/MobileIdentities.swift (hunk 2 above)
Comment (swarna04) [reply AEPIdentity/Sources/MobileIdentities.swift:52]: Yeah, that's how it was in the cpp implementation. I thought I'd keep these constants same to make it easier to refer back to. It's the same with `NAMESPACE_USERIDENTIFIER` here and `VISITOR_IDENTIFIER` in IdentityConstants for Analytics.

**Review by sbenedicadb — APPROVED** [review]

**Pull request merged**